src/source/temporalgrid_tile_worker_source.js

```diff
@@ -25,7 +25,10 @@ const getAggregationParams = params => {
         url.searchParams.get("quantizeOffset") || "0"
     );
     const singleFrame = url.searchParams.get("singleFrame") === "true";
+    // We want proxy active as default when api tiles auth is required
+    const proxy = url.searchParams.get("proxy") !== "false";
     const aggregationParams =  {
+        proxy,
```

Comment:
Can you explain why `aggregate` will need that parameter? (especially since I don't see it anywhere being used)

Reply:
You are right, it isn't used anywhere else but are the params we send to the getFinalUrl method, we could:

- Rename the `getAggregationParams` method
- Create another method with no `aggregation` params and merge them in https://github.com/GlobalFishingWatch/mapbox-gl-js/blob/temporalgrid-proxy/src/source/temporalgrid_tile_worker_source.js#L109

which one do you like most ?